### PR TITLE
Implementation of #100 -  Return to product grid button

### PR DESCRIPTION
**Pull request by ScoutAtlas**
#100

### CODE REVIEW

File: frontend/src/components/details/NavbarComponent.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-
 const NavbarComponent = (props) => {
+    const history = useHistory();
     if (!props.loadComparePage) {
         return (
-            <div className='navbar w3-row' vertical='center' horizontal='space-between'>
-                <Link to={{ pathname: slugs.categories }}>
-                    <btn className='w3-col l1 m1 s1 w3-center'>
+            <div className='navbar' vertical='center' horizontal='space-between'>
+               {/* used the history.goback() function to go one step backward where it stores the previous steps including the prevs stage of clicking items on the sidebar */}
```
Comment (Waldleufer) [inline frontend/src/components/details/NavbarComponent.js:17]: That's brilliant!

File: frontend/src/components/details/navbar.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
     height: auto;
     color: #1b1534;
     padding: 10px;
-    background-color: var(--global--button--color) !important;
+    background-color: #ffffff !important;
```
Comment (Waldleufer) [inline frontend/src/components/details/navbar.css:7]: I'd prefer having a variable for the color and the background-color. Is that possible?

File: frontend/src/components/details/navbar.css [hunk 3]
```diff
@@ -48,38 +48,39 @@
...
-    padding: 12px;
+    padding: 15px;
     /* color: var(--global--panel--color); */
     background-color: var(--global--button--color);
     color: var(--global--button--text--color);
     text-decoration: none;
     font-size: 22px;
-    width: 20%; /* Four equal-width links. If you have two links, use 50%, and 33.33% for three links, etc.. */
-    text-align: right; /* If you want the text to be centered */
+    width: 5%; /* Four equal-width links. If you have two links, use 50%, and 33.33% for three links, etc.. */
```
Comment (Waldleufer) [inline frontend/src/components/details/navbar.css:57]: I see ... this is how the proportions of the nav bar are defined.

**Review by Waldleufer — COMMENTED** [review]
:)

File: frontend/src/components/details/navbar.css (hunk 3 above)
Comment (Waldleufer) [reply frontend/src/components/details/navbar.css:57]: and maybe we can change that somewhat. In a way where not the whole whitespace is linking, and the buttons react as expected when resizing the page. https://github.com/amosproj/amos-ss2021-carbon-footprint/pull/123#discussion_r654427843

File: frontend/src/components/productGrid/CategoryIndex.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                    Meeting the growing global energy demand while transitioning to a more
+                    sustainable world requires a complex journey. But you're not without a partner.
+                    At Siemen's Carbon Visualization, we work closely with our customers to drive
+                    the energy transition, step by step, with our rich history of innovative
+                    technology,extensive energy experience and ambitious strategy to drive the
+                    decarbonization of global energy systems.
+                </p>
+            </div>
+            <div class='w3-row m6 32 w3-padding-small'>
+                <div class='w3-third imageTitle w3-animate-bottom'>
```
Comment (Waldleufer) [inline frontend/src/components/productGrid/CategoryIndex.js:22]: all these `class` should be `className` 🙂  @manifau 

have a look at the console (F12) There it says

image

🙂

**Review by Waldleufer — COMMENTED** [review]
🙂

File: frontend/src/components/productGrid/CategoryIndex.js (hunk 4 above)
Comment (manifau) [reply frontend/src/components/productGrid/CategoryIndex.js:22]: Thanks for pointing it out. I will change them :)

Comment (Waldleufer) [reply frontend/src/components/productGrid/CategoryIndex.js:22]: thank you. you are welcome :)

File: frontend/src/components/details/navbar.css (hunk 2 above)
Comment (IremToroslu) [reply frontend/src/components/details/navbar.css:7]: We can change that😄

File: frontend/src/components/details/navbar.css (hunk 3 above)
Comment (IremToroslu) [reply frontend/src/components/details/navbar.css:57]: we have already defined  the navbar in the back , I guess it is possible to resize the button sizes but lets see how that works at the end 😄

**Pull request merged**